website/docs/reference/widgets/text.md

```diff
@@ -1,94 +1,93 @@
-| **Visible**         | It controls the widget's visibility on the page. When turned off, the widget would not be visible when the app is published.                                                                |
-| **Animate Loading** | Control’s widget’s loading animation on the page. When turned off, the widget will load without any skeletal animation. This can be controlled with JS until all the widgets are rendered. |
-| **Disable link**    | It parses any link in the widget as standard text.                                                                                                                                         |
-| [**Height**](/reference/widgets/#height)        | It configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**Fixed**: The height of the widget remains as set using drag and resize.<br/>**Auto Height**: The height of the widget reacts to content changes.<br/>  **Auto Height with limits**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
+This method is useful when you need to display fixed information, such as **instructions** or **labels**, that don't change based on user interactions. 
 
-Let's understand the widget properties in detail.
+Additionally, you can use **HTML code** in the Text field to customize the appearance of the displayed text. Text field can only render inline CSS. To use external CSS, it's recommended to use the [Iframe widget](/reference/widgets/iframe). For instance, you can use:
 
-#### Text
-
-The `Text` field takes the input for the text widget. You can also add `HTML` code in the Text field manually or dynamically to render it in the widget.
-
-<VideoEmbed host="youtube" videoId="jIOajSSe6vI" title="Text" caption="Text"/>
+```js
+<p style="color:blue;">{{Input1.text}}</p>
+```
 
-:::info
-Text field can only render inline CSS. If you want to use external CSS, we suggest using the [iFrame widget.](iframe.md)
-:::
+## Display dynamic text
 
-#### Truncate Text
+To display dynamic text that changes based on user interactions or data from external sources, you can use binding values to connect different widgets and their properties.
 
-This property shortens the text in the text box and further adds three ellipses at the bottom left of the widget. Clicking on the three ellipses opens up a pop-up showing all the text inside the text widget. It's enabled by default and truncation will only be applied if the text is longer than what can fit inside the given widget area.
 
-<VideoEmbed host="youtube" videoId="Pex6RAyeHso" title="Truncate Text" caption="Truncate Text"/>
+For example, if you have a table widget with a list of tasks, and you want to display the details of the currently selected task in a text widget, you can use a binding like this:
```

Comment:
```suggestion
For example, if you have a Table widget with a list of tasks, and you want to display the details of the currently selected task in a Text widget:
```

Ensure that all widget names, properties etc are referred the was they are shown on the platform or in title case